1-js/11-async/03-promise-chaining/article.md

```diff
@@ -287,7 +287,7 @@ fetch('/article/promise-chaining/user.json')
   });
 ```
 
-The code works, see comments about the details. However, there's a potential problem in it, a typical error of those who begin to use promises.
+The code works; see comments about the details. However, there's a potential problem in it, a typical error of those who begin to use promises.
```

Comment:
Change OK. I would say "a typical error **for** those who begin..." ("for" rather than "of"). But small diff.